gpflow/training/scipy_optimizer.py

```diff
@@ -25,6 +25,32 @@ def __init__(self, **kwargs):
         self._optimizer_kwargs = kwargs
         self._optimizer = None
         self._model = None
+    
+
+    def make_optimizer_tensor(self, model, session, var_list=None, **kwargs):
+        """
+        Make SciPy optimization tensor.
+        The `make_optimizer_tensor` method builds optimization tensor and initializes
+        all necessary variables created by optimizer.
+
+            :param model: GPflow model.
+            :param session: Tensorflow session.
+            :param var_list: List of variables for training.
+            :param kwargs: Scipy optional optimization parameters,
+                - `maxiter`, maximal number of iterations to perform.
+                - `disp`, if True, prints convergence messages.
```

Comment:
It seems to return a tf.Tensor or a tf.Operation, and the same make_optimizer_operation would seem more natural to me (with the other method called make_optimizer_action). Just personal preferences, feel free to differ

Reply:
`*_tensor` refers to tensorflow, not a tensor as a structure.

Reply:
Then call it make_tensorflow_optimizer ?

Reply:
@st-- latest :)